src/main/java/org/terasology/weatherManager/systems/MarkovChainWeatherGenerator.java

```diff
@@ -156,7 +162,11 @@ public Vector2f nextWindCondition() {
         );
     }
 
-
+    /**
+     * Creates a new Weather Generator which generates random weather using Markov Chain
+     * @param seed the seed to be used for {@link FastRandom}
```

Comment:
One usually tries to hide implementation details like this. Instead of stating which random number generator (RNG) is used, you may describe the effect of the `seed`.

```suggestion
     * @param seed the seed to control the random weather generation
```

You could extend on this description in the comment text, maybe something like:

```
The <code>seed</code> controls the random weather generated. 
Using the same seed will yield the same sequence of pseudo-random weather conditions.
```

Reply:
Good point there. Will change it asap